code/modules/cargo/console.dm

```diff
@@ -168,6 +168,8 @@
 				log_game("[key_name(usr)] accepted a shuttle loan event.")
 				. = TRUE
 		if("add")
+			if(istype(src, /obj/machinery/computer/cargo/express)) //prevents running this in the wrong way
+				return . = FALSE
```

Comment:
`istype(src)` is almost never the right way to do things, what you want is a redefined proc
I assume you wanted to `return FALSE` so the rest of the proc doesn't happen.
The correct way h ere would be to move the things inside the `if("add)` check into a proc such as `/obj/machinery/computer/cargo/proc/add_id(id)`, which you redefine for `/obj/machinery/computer/cargo/express`